src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/layouts.prevalues.controller.js

```diff
@@ -62,13 +56,17 @@
          $scope.model.value.splice($index, 1);
       }
 
-      function openIconPicker(layout) {
+       function openIconPicker(layout) {
             var iconPicker = {
-                submit: function(model) {
-                    if (model.color) {
-                        layout.icon = model.icon + " " + model.color;
-                    } else {
-                        layout.icon = model.icon;
+                icon: layout.icon.split(' ')[0],
+                color: layout.icon.split(' ')[1],
```

Comment:
We need to make a 100% sure that there's always 2 values in that split value though?

To be safe, let's split it and if there's 1 item, check if it starts with `color-` or `icon-` and then you can assign the appropriate properties (`color:` or `icon:`). 
Can you update that please?

Reply:
@nul800sebastiaan I can have a look at that, but at the moment this is also how it works in icon picker in header :)
https://github.com/umbraco/Umbraco-CMS/blob/853087a75044b814df458457dc9a1f778cc89749/src/Umbraco.Web.UI.Client/src/common/directives/components/editor/umbeditorheader.directive.js#L229-L230

Reply:
I think it should be fine splitting it like this for now, since it already does it like this in e.g. document type header icon.

```
var layoutIcon = "icon-date";
var icon = layoutIcon.split(' ')[0],
		color = layoutIcon.split(' ')[1];
    
console.log("icon", icon); // returns "icon-date"
console.log("color", color);  // returns undefined
```

By default when specific color icon has been set, there is no `color-` class added, it just contain the class name for the icon, e.g. `icon-document`. It doesn't seems to break anything that it pass in `color: undefined` to the model as it already does in icon picker in header.